libextract/coretools.py

```diff
@@ -54,6 +54,15 @@ def mapper(etree):
     return decorator
 
 
+def get_node(pair):
+    """
```

Comment:
I am a little hesitant on leaving this in coretools, it is more implementation specific to article's strategy than generalised.